build-config/resources/Ballerina.toml

```diff
@@ -0,0 +1,31 @@
+[package]
```

Comment:
Do we need this file ? 🤔 If not let's remove.

Reply:
I followed this https://github.com/ballerina-platform/module-ballerina-http/blob/master/build-config/resources/Ballerina.toml